_layouts/default.html

```diff
@@ -8,10 +8,10 @@
 	<meta property="og:title" content="{%if page.title%}{{page.title}} | {%endif%}Debuggy"/>
 	<meta property="og:description" content="{{page.description}}"/>
 	{% if page.style %}
-	<link rel="stylesheet" href="/assets/css/{{page.style}}.css"/>
+	<link rel="stylesheet" href="assets/css/{{page.style}}.css"/>
 	{% endif %}
 	{% for style in page.styles %}
-	<link rel="stylesheet" href="/assets/css/{{style}}.css"/>
+	<link rel="stylesheet" href="assets/css/{{style}}.css"/>
```

Comment:
More so a note of if we ever start nesting pages:

We might want to look into having Jekyll generate relative paths correctly so nesting can work no matter how this is deployed.

It originally referenced root as it was intended to be deployed to root level.